pom.xml

```diff
@@ -87,6 +87,7 @@
         <module>vaadin-rich-text-editor-flow-parent</module>
         <module>vaadin-spreadsheet-flow-parent</module>
         <module>vaadin-field-highlighter-flow-parent</module>
+        <module>vaadin-side-nav-flow-parent</module>
```

Comment:
BTW, we might want to sort these alphabetically at some point. I will create a PR.